packages/@o3r/application/schemas/cms.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "CmsMetadataSchema",
+  "type": "object",
+  "title": "Schema of CMS application configuration",
+  "description": "Schema used to indicate the configuration to the CMS when the application is loaded by it",
+  "additionalProperties": false,
+  "properties": {
+    "assetsFolder": {
+      "type": "string",
+      "description": "The relative path to the assets folder to import",
+      "default": "dist/assets"
+    },
+    "localisationsFolder": {
```

Comment:
```suggestion
    "localizationsFolder": {
```

Reply:
This is the field as specified in the doc